staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go

```diff
@@ -190,6 +202,19 @@ func (w *WebhookAuthorizer) Authorize(ctx context.Context, attr authorizer.Attri
 			Verb: attr.GetVerb(),
 		}
 	}
+	// Process Match Conditions before calling the webhook
+	matches, err := w.Match(ctx, r)
```

Comment:
When the feature gate is disabled, we should skip processing the match conditions.

Reply:
added